src/index.html

```diff
@@ -9,6 +9,93 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="nav__div">
```

Comment:
```suggestion
      <div class="header__image-wrapper">
```
it's not part of `nav`
`div` is bad element name